standalone/src/models/attribute-group/attribute-group-draft/fields-config.ts

```diff
@@ -14,7 +14,7 @@ import type {
 const commonFieldsConfig = {
   key: null,
   description: null,
-  name: fake(() => LocalizedString.random()),
+  name: fake(() => LocalizedStringDraft.random()),
```

Comment:
This made the resulting object to contain a `__typename` property which is problematic for consumers.